mdakits/membrane-curvature/metadata.yaml

```
@@ -0,0 +1,117 @@
+# TEMPLATE MDAKit file
+# --------------------
+# 
+# Please replace ALL entries with appropriate content for YOUR MDAKit.
+# Below we use the placeholder GH_HOST_ACCOUNT for the GitHub account where
+# the source code repository is held, typically your username or the 
+# organization that you're part off.
+# MYPROJECT is the name of your project (the repository name and here
+# we assume that this is also the PyPi/conda package name) whereas 
+# MYPACKAGE is how you import it in python.
+#
+# See https://mdakits.mdanalysis.org/add.html for more information. 
+#
+#------------------------------------------------------------
+# Required entries
+#------------------------------------------------------------
+## str: name of the project (the respository name)
+project_name: membrane-curvature
+
+## List(str): a link to the authors file (preferred) or a list of authors 
+authors:
+  - Estefania Barreto-Ojeda
```

I'd suggest pointing to the AUTHORS file here instead:
```suggestion
  - https://github.com/MDAnalysis/membrane-curvature/blob/main/AUTHORS
```